alacritty_terminal/src/display.rs

```diff
@@ -368,7 +368,10 @@ impl Display {
             }
         }
 
-        if font_changed || self.last_message != terminal.message_buffer_mut().message() {
+        // Message bar message update detected
+        let message_changed = self.last_message != terminal.message_buffer_mut().message();
```

Comment:
I think being a little bit more verbose here would be beneficial. So `message_bar_changed` would imho be the better variable name. It's easy to get confused when talking about messages, so when possible we should not skip the full name here.

The line should also stay under 100 columns, so it's just a little bit more verbose without any formatting disadvantages.